src/mkdocstrings/handlers/base.py

```diff
@@ -577,7 +577,7 @@ def get_handler(self, name: str, handler_config: Optional[dict] = None) -> BaseH
 
         Returns:
             An instance of a subclass of [`BaseHandler`][mkdocstrings.handlers.base.BaseHandler],
-            as instantiated by the `get_handler` method of the handler's module.
+                as instantiated by the `get_handler` method of the handler's module.
```

Comment:
Because of this :)

![image](https://user-images.githubusercontent.com/30731072/214604098-dea8536a-3ad6-4dc5-9ec1-59bd8c4e2a40.png)